### PR TITLE
Allow for colons in GOPATH, allow for spaces in GOPATH and current directory

### PR DESCRIPTION
**Pull request by jethrogb**

### CODE REVIEW

File: go/run/scripts/build_docker.sh [hunk 1]
```diff
@@ -16,23 +16,25 @@ function build_docker() {
 	# This script currently only supports Linux (or any system that has a working
 	# readlink -e)
 
-	script_name=$1
+	script_name="$1"
```
Comment (tmroeder) [inline go/run/scripts/build_docker.sh]: For consistency, could you please quote the other $N arguments when you quote one of them? E.g., it happens that policy_cert ($3) and tao_config ($4) are passed from as paths without spaces, but that's a fragile assumption (one that I made, of course).

**Comment by tmroeder** [comment]
Thanks very much for catching this. I hadn't thought about the case of spaces in the path or in GOPATH. I have a minor request to fix the other $N arguments for consistency.

I have tried this patch on my system, and it works with all of process/docker/KVM using Datalog auth.

**Comment by tmroeder** [comment]
I'll add the quotes around the $N args after merging the pull request.

**Comment by jethrogb** [comment]
Thanks.

**Pull request merged**